types/sharepoint/index.d.ts

```diff
@@ -25,6 +25,100 @@ declare const enum Sods {
     loaded = 4,
     error = 5
 }
+
+declare function GetThemedImageUrl(image: string): string;
```

Comment:
shouldn't be this method in the Utility class? https://docs.microsoft.com/en-us/dotnet/api/microsoft.sharepoint.utilities.sputility.getthemedimageurl?view=sharepoint-server

Reply:
Not on the declarations I have. It gets pulled from init.js and is not scoped from what I can see. The link you pointed to seems to refer to SSOM, not the JS library.

Reply:
ok thanks I was simply double checking!